tests/accessor/generic_accessor_properties_fp16.cpp

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for generic accessor properties with sycl::half type
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+
+// FIXME: re-enable when sycl::accessor is implemented
+#if !defined(__HIPSYCL__) && !defined(__COMPUTECPP__) && \
+    !defined(__SYCL_COMPILER_VERSION)
+#include "accessor_common.h"
+#include "generic_accessor_properties.hpp"
```

Comment:
Looks like we've started adding `.hpp` files at some point in tests for accessors. Let's align with the rest of the code base and use `.h` extension. Currently we don't even have `clangformat.py` enabled for `hpp` files.

Reply:
Renamed